lib/chore/new_relic.rb

```diff
@@ -1,4 +1,4 @@
-gem 'newrelic_rpm', '>= 3.7.0'
+gem 'newrelic_rpm', '>= 3.15.0'
```

Comment:
Reviewers: Can we move this into the gemspec, or is there a specific reason we're doing it like this?